spec/services/pii/signing_key_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+describe Pii::SigningKey do
+  describe '#new' do
+    it 'automatically loads the server-wide signing key' do
+      expect(subject.rsa_key).to be_a OpenSSL::PKey::RSA
+    end
+  end
+
+  describe '#sign' do
+    it 'signs text with private key' do
+      signature = subject.sign('foo')
+
+      expect(signature).to be_a String
```

Comment:
is there a more specific thing we could be looking for? Maybe `expect(signature).not_to eq 'foo'` ?